test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -559,6 +559,24 @@ test
     )).eql(2);
   });
 
+test
+  .requestHooks(customURILogger, customURIMock)('in custom URI approach, Okta Verify iframe should be single and hidden', async t => {
+    const deviceChallengePollPageObject = await setup(t);
+    await t.wait(100); // opening the page in iframe takes just a moment
```

Comment:
Is there a way to wait for a specific element to appear, rather than using `t.wait`

Reply:
Changed to wait for iframe